### DISCUSSION TITLE
is Cocopad is compatiable with Patrol?

### DISCUSSION THREAD

**Discussion by akagupta9** [leancodepl/patrol#810] [Q&A]
I am getting build failure for IOS -

CocoaPods could not find compatible versions for pod "patrol"

**Comment by bartekpacia** [comment]
This looks like a bug, please share more info, otherwise, I cannot help you resolve the problem.

**Reply by akagupta9** [reply]
[!] CocoaPods could not find compatible versions for pod "patrol":
      In Podfile:
        patrol (from `.symlinks/plugins/patrol/ios`)

    Specs satisfying the `patrol (from `.symlinks/plugins/patrol/ios`)` dependency were found, but they required a higher minimum deployment target.

**Reply by akagupta9** [reply]
what minimum version required?

**Reply by bartekpacia** [reply]
You need to have minimum iOS set to 13.0 in your `Podfile`. This should be noted somewhere in documentation, sorry.

**Reply by maxbritto** [reply]
Thanks for the feedback @bartekpacia !
So, should we also restrict the minimum iOS version to 13 in our Xcode project file ? This means restricting user devices compatibility for a dev dependency that never lands on their phone 🤔

**Reply by bartekpacia** [reply]
@maxbritto Unfortunately, yes. We're researching better ways to solve it but so far, we haven't found anything.

**Reply by maxbritto** [reply]
Thanks for the quick response and the amazing work on patrol. 
For now I'll revert back the tests I wrote with Patrol and might come back once we decide to move to iOS version 13 and Android version 21 as minimum deployment target.
